pages/configuration/resin-base-images.md

```diff
@@ -1,8 +1,65 @@
 # Resin Images Wiki
 
-This wiki page contains all information about images on Resin registry.
+This wiki page contains all information about images on Resin registry. 
```

Comment:
Should be 
"This page contains all the information about the image maintained on the Resin.io docker hub registry."